src/ipc/Kid.cc

```diff
@@ -11,6 +11,8 @@
 #include "squid.h"
 #include "globals.h"
 #include "ipc/Kid.h"
+#include "SquidConfig.h"
+#include "SquidTime.h"
```

Comment:
I am not against this change, but, for the record, we should not expect humans to minimize `#include` lists. Including file X in file Y when X declares something used in Y is fine, regardless of whether Y also includes some other file Z which also includes X.

IMHO, in general, the list of includes in Z is not a part of Z's public API -- it provides no guarantees and may change at any time. For example,  Z might include X to declare some private Z class member; changes in such private declarations ought to have no effect on Z users.